scripts/bash/s3a_wrapper.sh

```diff
@@ -105,6 +106,18 @@ function cleanup_versions() {
   return 0
 }
 
+# Function to get evaluated tmp path for standardization
+# Replace placeholders if present
+function replace_std_path_placeholders() {
```

Comment:
Is this correct?
The function is more like a procedure and used as a function.
Also is the use of `$path` correct, not `$std_path`?

Reply:
It is fine. Bash has no concept of procedures. And this way I can nicely chain it. 

Path fixed